packages/unmock-xmlhttprequest/src/__tests__/response-function.test.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @jest-environment jsdom
+ */
+import {
+  ISerializedRequest,
+  ISerializedResponse,
+  OnSerializedRequest,
+} from "unmock-core/dist/interfaces";
+import { replaceOpenAndReturnOriginal } from "..";
+
+/**
+ * Example algorithm for how to calculate the response from serialized request.
+ * @param req Serialized request
+ * @param sendResponse Function for sending the response, handed by the interceptor.
+ * @param emitError Function for emitting an error, handed by the interceptor.
+ */
+const respondOk: OnSerializedRequest = (
+  _: ISerializedRequest,
+  sendResponse: (res: ISerializedResponse) => void,
+  emitError: (e: Error) => void,
+) => {
+  try {
+    const res: ISerializedResponse = {
+      headers: {},
+      statusCode: 200,
+      body: JSON.stringify({
+        ok: true,
+      }),
+    };
+    sendResponse(res);
+  } catch (err) {
+    emitError(err);
+  }
+};
+
+replaceOpenAndReturnOriginal(respondOk);
```

Comment:
I think this would persist to other tests if running in band? Would it be safer to switch it off at the end of the test?